src/ViewLayer/Faults/MotorFaults/MotorFaultList.cpp

```diff
@@ -34,13 +34,20 @@ FaultDisplayData MotorFaultList::nextActiveFault()
     return FaultDisplayData();
 }
 
+QVector<QString>& MotorFaultList::activeFaultLabels()
```

Comment:
All comments I made in BatteryFaultList.cpp apply here.

Reply:
Matter of fact, I remember wanting to bring this up since I had to read the code in Feb.

BatteryFaultList.cpp vs MotorFaultList.cpp and their related files, are very similar. Should we create something like a FaultList base class?